onnxruntime/contrib_ops/cuda/cuda_contrib_kernels.cc

```diff
@@ -17,6 +17,9 @@ class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCudaExecutionProvider, kMSDomain, 1
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCudaExecutionProvider, kMSDomain, 1, float, BiasGelu);
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCudaExecutionProvider, kMSDomain, 1, double, BiasGelu);
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCudaExecutionProvider, kMSDomain, 1, MLFloat16, BiasGelu);
+class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCudaExecutionProvider, kMSDomain, 1, float, BiasSoftmax);
```

Comment:
consider using MLTypeCallDispatcher to avoid having a separate kernel per type.
https://github.com/microsoft/onnxruntime/blob/c6a3620ba8ebfd1e199eeb01f33d1419df40b337/include/onnxruntime/core/framework/data_types_internal.h#L300